site/content/3.12/get-started/start-using-aql/_index.md

```diff
@@ -0,0 +1,156 @@
+{{< /tab >}}
+
+{{< tab "arangosh" >}}
+You can run AQL queries from the ArangoDB Shell (arangosh)
+with the [`db._query()`](../../aql/how-to-invoke-aql/with-arangosh.md#with-db_query) and
+[`db._createStatement()`](../../aql/how-to-invoke-aql/with-arangosh.md#with-db_createstatement-arangostatement)
+methods of the [`db` object](../../develop/javascript-api/@arangodb/db-object.md).
+
+If you use Foxx, see [how to write database queries](../../develop/foxx-microservices/getting-started.md#writing-database-queries)
+for examples including tagged template strings.
+
+```js
+---
+name: arangosh_execute_query_bindvars
+description: ''
+---
+db._query(`RETURN CONCAT("Hello, ", @name)`, { name: "AQL" }).toArray();
+// -- or --
+var name = "AQL";
+db._query(aql`RETURN CONCAT("Hello, ", ${name})`).toArray();
+```
+See [`db._query()`](../../develop/javascript-api/@arangodb/db-object.md#db_queryquerystring--bindvars--mainoptions--suboptions)
+in the _JavaScript API_ for details.
+{{< /tab >}}
+
+{{< tab "cURL" >}}
+You can use a tool like [cURL](https://curl.se/) to run AQL queries from a
+command-line, directly using the HTTP REST API of ArangoDB.
+
+The response bodies are generally compact JSON (without any line breaks and
+indentation). You can format them with the [jq](https://jqlang.github.io/jq/)
+tool for better readability if you have it installed:
+
+```sh
+curl -d '{"query":"RETURN CONCAT(\"Hello, \", @name)","bindVars":{"name":"AQL"}}' http://localhost:8529/_api/cursor | jq
+```
+
+See the [`POST /_db/{database-name}/_api/cursor`](../../develop/http-api/queries/aql-queries.md#create-a-cursor)
+endpoint in the _HTTP API_ for details.
+{{< /tab >}}
+
+{{< tab "JavaScript" >}}
+```js
+import { Database } from "arangojs";
+const db = new Database();
+
+const name = "AQL";
+const result = await db.query(aql`RETURN CONCAT("Hello, ", ${name})`);
+console.log(result);
+```
+
+See [`Database.query()`](https://arangodb.github.io/arangojs/latest/classes/database.Database.html#query)
+in the _arangojs_ documentation for details.
+{{< /tab >}}
+
+{{< tab "Go" >}}
+```go
+ctx := context.Background()
+query := `RETURN CONCAT("Hello, ", @name)`
+options := arangodb.QueryOptions{
+    BindVars: map[string]interface{}{
+        "name": "AQL",
+    }
+}
+cursor, err := db.Query(ctx, query, &options)
+if err != nil {
+    // handle error
+} else {
+    defer cursor.Close()
+    var doc map[string]interface{} 
+    for cursor.HasMore() {
+        meta, err = cursor.ReadDocument(ctx, &doc)
+        if err != nil {
+            // handle error
+        } else {
+            fmt.Printf("%+v\n", doc)
+        }
+    }
+}
+```
+
+See [`DatabaseQuery.Query()`](https://pkg.go.dev/github.com/arangodb/go-driver/v2/arangodb#DatabaseQuery)
+in the _go-driver_ v2 documentation for details.
+{{< /tab >}}
+
+{{< tab "Java" >}}
+```java
+String query = "RETURN CONCAT(\"Hello, \", @name)";
+Map<String, Object> bindVars = Collections.singletonMap("name", "AQL");
```

Comment:
This is an error at least with Java 8:
> Type mismatch: cannot convert from Map<String,String> to Map<String,Object>

`Map<String, String>` OTOH is not accepted by the query() method. The only working type I found so far is `Map<String, ? super Object>`...